src/engine.h

```diff
@@ -131,10 +134,12 @@ class EngineLoop : public UciLoop {
   void CmdUciNewGame() override;
   void CmdPosition(const std::string& position,
                    const std::vector<std::string>& moves) override;
+  void CmdGetFen();
```

Comment:
This is not needed anymore.

Reply:
This is needed again...
Sorry for making to do and undo extra work, I've just took a look what it would take to add this "NonStandardUciCommand" function cleanly and it really doesn't seem to be that clean.